.rubocop.yml

```diff
@@ -1,24 +1,28 @@
 inherit_from: .rubocop_todo.yml
 
 AllCops:
-  TargetRubyVersion: 2.2
+  TargetRubyVersion: 3.0
+  NewCops: enable
 
-ClassLength:
-  Max: 200
+Gemspec/RequiredRubyVersion:
```

Comment:
disabled this one because it was complaining that the required ruby version is lower than the target, but raising the required would remove backwards compatibility with ruby 2 versions...

Reply:
it looks like the target is supposed to be the oldest supported version...
https://docs.rubocop.org/rubocop/configuration.html#setting-the-target-ruby-version